pimd/pim_zebra.c

```diff
@@ -119,13 +119,18 @@ static int pim_zebra_if_add(int command, struct zclient *zclient,
 		pim_sock_add(ifp);
 	}
 
+	if (!strncmp(ifp->name, PIM_VXLAN_TERM_DEV_NAME,
+				sizeof(PIM_VXLAN_TERM_DEV_NAME)))
```

Comment:
From commit msg :"The default termination device is expected to have the sub-string
"ipmr-lo". This can be made configurable if needed". 

My thought was even if the device name is not configurable, I can still create one with name "my-ipmr-lo" and it should still work but that is not the case. the device name should start with string "ipmr-lo" or else it wont work, i.e, not just a sub-string

Reply:
That is true. I will fixup the commit message to say that it should include start with prefix ipmr-lo.